generic-examples/kafka/sasl_ssl/SaslSSLKafkaProducer.java

```diff
@@ -23,12 +23,12 @@
 public class SaslSSLKafkaProducer extends RouteBuilder {
   @Override
   public void configure() throws Exception {
-  log.info("About to start route: Timer -> Kafka ");
-  from("timer:foo")
-    .routeId("FromTimer2Kafka")
-    .setBody()
-      .simple("Message #${exchangeProperty.CamelTimerCounter}")
-	  .to("kafka:{{producer.topic}}")
-    .log("Message correctly sent to the topic!");
+    log.info("About to start route: Timer -> Kafka ");
+    from("timer:foo")
```

Comment:
This indentation can be better. Can we have something like this?
```
    from("timer:foo")
        .routeId("FromTimer2Kafka")
        .setBody()
            .simple("Message #${exchangeProperty.CamelTimerCounter}")
        .to("kafka:{{producer.topic}}")
        .log("Message correctly sent to the topic!");
```

Reply:
Done